resources/js/Components/twc-ui/hover-card.tsx

```
@@ -0,0 +1,33 @@
+'use client'
+
+import * as React from 'react'
```

Biome (2.3.13)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Unsafe fix: Remove the unused imports.


(lint/correctness/noUnusedImports)